front-end/src/components/Flicker/Flicker.styled.ts

```diff
@@ -0,0 +1,59 @@
+import styled, { css, keyframes } from "styled-components";
+
+type FlickerStatus = "ON" | "WARNING";
+type Size = "sm" | "md" | "lg";
+
+export interface StyledFlickerProps {
+  status: FlickerStatus;
+  size: Size;
+}
+
+const flick = keyframes`
+  0% { opacity: 0; }
+  100% { opacity: 1; }
+`;
+
+const getFlickerStatus = (status: FlickerStatus) => {
+  const flickerType = {
+    // TODO: flicker color 상수화
+    ON: css`
+      background-color: #90dd53;
+      animation: ${flick} 0.5s ease-in-out infinite alternate;
+    `,
+    WARNING: css`
+      background-color: #f86565;
+      animation: ${flick} 0.3s ease-in-out infinite alternate;
+    `,
+  } as const;
+
+  return flickerType[status];
+};
+
+const getFlickerSize = (size: Size) => {
+  const flickerSize = {
+    // TODO: flicker color 상수화
+    sm: css`
+      width: 1rem;
+      height: 1rem;
+    `,
+    md: css`
+      width: 1.5rem;
+      height: 1.5rem;
+    `,
+    lg: css`
+      width: 2.5rem;
+      height: 2.5rem;
+    `,
+  } as const;
+
+  return flickerSize[size];
+};
```

Comment:
다른 스타일드 컴포넌트에서는 get...style 로 함수를 작성했는데, 비슷하게 맞추면 어떨까요?

아니면 다른 스타일드 컴포넌트를 여기다 맞춰도 좋구요. 

개인적으로는 함수명 마지막에 style 또는 css 가 있었으면 좋을 것 같아요 .
status 에 대한 style을 가져오는 것이지, 해당 status를 가지고 오는건 아니니까요

```suggestion
const getStatusStyle = (status: FlickerStatus) => {
  const style = {
    // TODO: flicker color 상수화
    ON: css`
      background-color: #90dd53;
      animation: ${flick} 0.5s ease-in-out infinite alternate;
    `,
    WARNING: css`
      background-color: #f86565;
      animation: ${flick} 0.3s ease-in-out infinite alternate;
    `,
  } as const;

  return style[status];
};

const getSizeStyle = (size: Size) => {
  const flickerSize = {
    // TODO: flicker color 상수화
    sm: css`
      width: 1rem;
      height: 1rem;
    `,
    md: css`
      width: 1.5rem;
      height: 1.5rem;
    `,
    lg: css`
      width: 2.5rem;
      height: 2.5rem;
    `,
  } as const;

  return flickerSize[size];
};
```

Reply:
`styled` 관련 `prop`에 대해서 `get...Style` 형태로 가져가겠습니다 :)